ui/packages/catalog-ui-search/src/main/webapp/component/map-info/map-info.view.js

```diff
@@ -62,8 +63,21 @@ module.exports = Marionette.LayoutView.extend({
     this.$el.toggleClass('has-feature', this.model.get('target') !== undefined)
   },
   serializeData: function() {
+    const that = this
     let modelJSON = this.model.toJSON()
+    let summaryModel = {}
+    const summary = props.summaryShow
+    summary.forEach(function(attribute) {
```

Comment:
Should probably limit the number of configurations that can be added to avoid any weird UI issues from having too many attributes being displayed inside Intrigue.